deploy/ingress-controller.yaml.example

```diff
@@ -18,10 +18,15 @@ spec:
       labels:
         application: kube-ingress-aws-controller
         component: ingress
+      annotations:
+        iam.amazonaws.com/role: arn:aws:iam::12345678901:role/YOUR_PRECONFIGURED_ROLE
     spec:
+      serviceAccountName: kube-ingress-aws
       containers:
       - name: controller
-        image: registry.opensource.zalan.do/teapot/kube-ingress-aws-controller:latest
+        image: registry.opensource.zalan.do/teapot/kube-ingress-aws-controller:v0.7.4
```

Comment:
I think it's better to use `latest` here, otherwise we will just forget to update this and it will get outdated. WDYT @szuecs ?

Reply:
Our registry has no latest tag that is updated

Reply:
`latest` tag automatically resolves to the latest pushed version.

```sh
$ docker pull registry.opensource.zalan.do/teapot/kube-ingress-aws-controller:latest
latest: Pulling from teapot/kube-ingress-aws-controller
8e3ba11ec2a2: Already exists 
fed9443358de: Already exists 
c6cf09988571: Pull complete 
Digest: sha256:e596f90a7a6bab308fc7e177d39b77516a5b88ddefcd2679bcbe670c12c94a69
Status: Downloaded newer image for registry.opensource.zalan.do/teapot/kube-ingress-aws-controller:latest
```

Reply:
ok then I just tricked myself with using `docker run`...

Reply:
In your docs you say that the latest tag is not maintained.

Reply:
@zioalex we should clean this up. Would you mind doint this for us?

Reply:
@szuecs what do you mean? The doc or the tag?

Reply:
The tag and clean the docs that you found telling that latest is not maintained.

Reply:
I update the tag but I cannot find it anymore in the doc.